packages/tracing/src/traceDatabaseCalls.ts

```diff
@@ -0,0 +1,49 @@
+import { trace, context } from '@opentelemetry/api';
+import type { MongoClient } from 'mongodb';
+
+import { isTracingEnabled } from '.';
+
+const tracer = trace.getTracer('core');
```

Comment:
Also, I'd not hardcode 'core' here, even if we are sharing the DB among the services.